decidim-core/app/views/decidim/pages/_standalone.html.erb

```diff
@@ -1,21 +1,17 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
-        <%= title %>
-      </h1>
-    </div>
+<%= render layout:"layouts/decidim/shared/layout_center" do %>
 
-    <%= cell "decidim/tos_page", :announcement %>
+  <div class="text-center py-12">
+    <h1 class="h1 decorator inline-block text-left">
+      <%= title %>
+    </h1>
+  </div>
 
-    <div class="columns small-12">
-      <div class="card">
-        <div class="card__content"><%= decidim_sanitize_editor translated_attribute page.content %></div>
-      </div>
-    </div>
+  <%= cell "decidim/tos_page", :announcement %>
 
-    <div class="columns small-12">
-      <%= cell "decidim/tos_page", :form %>
-    </div>
+  <div class="editor-content">
+    <%= decidim_sanitize_editor translated_attribute page.content %>
   </div>
-</div>
+
+  <%= cell "decidim/tos_page", :form %>
```

Comment:
Same thing when we have the callout and TOS acceptance form, this page is very cluttered now both on desktop and mobile.

**Mobile**
![Cluttered TOS page on mobile](https://user-images.githubusercontent.com/864340/184160491-cc68b3ea-9741-449b-ac12-e264d75ee661.png)

**Desktop**
![Cluttered TOS page on desktop](https://user-images.githubusercontent.com/864340/184160411-4bc4aa3c-b735-444d-b937-27f331e8d52b.png)

Reply:
@Crashillo I had another look but this issue was not addressed.

Now the standalone page has more spacing by default but with the announcement and the TOS form, it is still cluttered as in the screenshots above.

Could you have another look at this?